openreview/venue_request/process/deployProcess.py

```diff
@@ -85,7 +93,7 @@ def process(client, note, invitation):
 
 If you accept, please make sure that your OpenReview account is updated and lists all the emails you are using.  Visit http://openreview.net/profile after logging in.
 
-If you have any questions, please contact info@openreview.net.
+If you have any questions, please contact ''' + contact_email + '''.
```

Comment:
use `{contact_info}` instead.

Reply:
Once you add the parameter contact_info in conference.recruit_reviewers, you need to pass this parameter here:

https://github.com/openreview/openreview-py/blob/master/openreview/venue_request/process/recruitmentProcess.py#L38

you can verify this is correctly working in the test_neuirps_conference.py, making sure the content of the email has the right contact_info value.